charts/app-config/values-production.yaml

```diff
@@ -2590,6 +2590,30 @@ govukApplications:
 
   - name: smokey
     chartPath: charts/smokey
+    helmValues:
+      cronJobs:
+        - name: smokey
+          schedule: "*/10 * * * *"
+          args:
+            - cucumber
+            - --profile={{ .Values.govukEnvironment }}
+            - --strict-undefined
+            - --tags=not @notreplatforming and not @not{{ .Values.govukEnvironment }}
+        - name: smokey-failover
+          schedule: "0 9 * * 1-5"
+          args:
+            - env GOVUK_PROXY_PROFILE=failoverCDN
+            - ruby govuk_proxy_runner.rb &
```

Comment:
Does this work? It looks a bit like shell script but isn't gonna be interpreted as such. You can either fork/exec this govuk_proxy_runner from Smokey (based on whether `GOVUK_PROXY_PROFILE` is non-empty`) or define a separate container for the proxy (by changing the template to let you pass it in from here).